### PR TITLE
config: add lifecycle-periodics.yaml

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ thanks!

File: prow/jobs/ti-community-infra/lifecycle-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      volumes:
+        - name: github-token
+          secret:
+            secretName: github-token
+
+  - name: periodic-ti-community-infra-stale
+    interval: 1h
+    decorate: true
+    annotations:
+      description: Adds lifecycle/stale to issues after 30d of inactivity
```
Comment (Mini256) [inline prow/jobs/ti-community-infra/lifecycle-periodics.yaml:78]: The description here seems to be inconsistent with the comment of the later parameter.

Comment (hi-rustin) [reply prow/jobs/ti-community-infra/lifecycle-periodics.yaml:78]: I don't get what do you mean?

File: prow/jobs/ti-community-infra/lifecycle-periodics.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              -label:lifecycle/rotten
+            - --updated=2160h
+            - --token=/etc/github/token
+            - |-
+              --comment=Issues go stale after 90d of inactivity.
+              Mark the issue as fresh with `/remove-lifecycle stale`.
+              Stale issues rot after an additional 30d of inactivity and eventually close.
+              If this issue is safe to close now please do so with `/close`.
+              Send feedback to sig-community-infra or [hi-rustin](https://github.com/hi-rustin).
+              /lifecycle stale
```
Comment (hi-rustin) [inline prow/jobs/ti-community-infra/lifecycle-periodics.yaml:98]: I think it's right.

Comment (Mini256) [reply prow/jobs/ti-community-infra/lifecycle-periodics.yaml:98]: The description is `Adds lifecycle/stale to issues after 30d of inactivity`， but the comment is `Issues go stale after 90d of inactivity.`.
So how many days does it take to be inactive for an issue to become stale?

Comment (hi-rustin) [reply prow/jobs/ti-community-infra/lifecycle-periodics.yaml:98]: Good catch! Fixed.

**Review by Mini256 — APPROVED** [review]

**Comment by hi-rustin** [comment]
@Mini256  thanks!

**Pull request merged**